src/main/java/uk/gov/justice/probation/courtcasematcher/model/MessageAttributes.java

```diff
@@ -5,12 +5,11 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import uk.gov.justice.probation.courtcasematcher.messaging.model.MessageType;
-import uk.gov.justice.probation.courtcasematcher.model.type.HearingEventType;
 
 @Data
 @AllArgsConstructor
 @NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
 public class MessageAttributes {
     private final MessageType messageType;
-    private final HearingEventType hearingEventType;
+    private final String hearingEventType;
```

Comment:
This won't work, each SNS message attribute has the following structure:
```
{ Type: "String", Value: "Resulted" }
```

That's why we needed this line in the `HearingEventType` Enum to take the value from the "Value" attribute rather than attempt to deserialise the object directly:
```
@JsonCreator
    public static HearingEventType of(@JsonProperty("Value") String eventTypeDescription)
```

Here's an example of how attributes come across: 

```
    "messageType" : {"Type":"String","Value":"LIBRA_COURT_CASE"}
```